src/lib/librnp.3.adoc

```diff
@@ -0,0 +1,98 @@
+= librnp(3)
+RNP
+:doctype: manpage
+:release-version: 0.14.0
+:man manual: RNP Manual
+:man source: RNP {release-version}
+
+== NAME
+
+librnp - OpenPGP implementation, available via FFI interface.
+
+== SYNOPSIS
+
+*#include <rnp/rnp.h>* +
+*#include <rnp/rnp_err.h>*
+
+
+== DESCRIPTION
+
+*librnp* is part of the *RNP* suite and forms the basis for the _rnp(1)_ and _rnpkeys(1)_ command-line utilities.
+
+It provides an FFI interface to functions required for operations needed by the OpenPGP protocol.
+
+Interface to the library is exposed via _<rnp/rnp.h>_ and _<rnp/rnp_err.h>_ headers.
+You will also need to link to _librnp_.
+
+Please see its headers for the full function list and detailed documentation.
+
+
+== OPTIONS
+
+*-o, --out-file*=_OUT_FILE_::
+  Write result to file _OUT_FILE_.
+
+*-c, --capture*::
+  Capture specimen if it's a picture of a life form.
+
```

Comment:
@ronaldtse Does this came here by mistake or by intention :-) ?

Reply:
Complete mistake. Sorry. Can you help fix it?

Reply:
Thank you for spotting...

Reply:
@ronaldtse Sure.